src/electron/views/device-connect-view/components/window-title.scss

```
@@ -4,7 +4,23 @@
 @import '../../../../common/styles/fonts.scss';
 
 .window-title {
-    padding: 6px 9px;
+    width: 100%;
+    height: 32px;
+    display: flex;
+    align-items: center;
+
+    .title-container {
+        padding: 6px 9px;
+        display: flex;
+    }
+
+    .actionable-icons-container {
+        margin-left: auto;
```

```suggestion
        margin-left: auto;
        -webkit-app-region: no-drag;
```

If you take the other suggestion, this should accompany it to make sure the buttons still function.